code/modules/mob/living/carbon/xenomorph/embryo.dm

```diff
@@ -169,6 +169,9 @@
 								"<span class='danger'>You feel something ripping up your insides!</span>")
 	victim.jitter(300)
 
+	var/preburst_screams = victim.gender == FEMALE ? get_sfx("female_preburst") : get_sfx("male_preburst")
+	playsound(loc, preburst_screams, 25, 0)
+
```

Comment:
Instead of removing the *scream, could you not work these sounds into those screams?

Reply:
they already have screaming in them. Also having the *scream and the larva burst sound play over eachother doesn't work ingame that well, since you can't really hear both